.rultor.yml

```diff
@@ -1,5 +1,9 @@
 install:
   - if ! pip3 -V ; then wget -O - https://bootstrap.pypa.io/get-pip.py | python3; fi
+  - sudo python setup.py build
+  - sudo python setup.py install
+  - sudo apt-get install espeak libclang1-3.4 python3-gi python3-dbus -y
+  - pip3 install setuptools twine wheel munkres3 coverage pylint language-check pytest tox appdirs
```

Comment:
shouldn't you rather install the stuff from requirements.txt?

Also we don't need espeak and clang e.g. those are only needed for coala=bears right?